hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -1,16 +1,19 @@
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
 
-import logging
 import os
 import bottle
 import boto3
 import json
 import base64
+from dataclasses import dataclass, asdict
+import typing as t
 from boto3.dynamodb.conditions import Attr
 from apig_wsgi import make_lambda_handler
 from bottle import response, error
 
+
 from hmalib.common import get_logger
+from hmalib.models import PDQMatchRecord, PipelinePDQHashRecord
```

Comment:
nit w/naming... I personally prefer omitting prefixes / suffixes like `Record` - maybe drop it? It's like putting in a type in a name, for me, I guess.